libobs/obs.c

```diff
@@ -779,6 +779,14 @@ static bool obs_init(const char *locale, const char *module_config_path,
 extern void initialize_crash_handler(void);
 extern void initialize_com(void);
 extern void uninitialize_com(void);
+
+void obs_init_crash_handler(void) {
+	initialize_crash_handler();
+}
+#else
+void obs_init_crash_handler(void) {
```

Comment:
Style:  The opening curly brace for a function should be on the line after the parameters.